index.js

```diff
@@ -1,88 +1,128 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1
+counter++
+
+console.log(counter) 
+//=> 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+const name = "Jaime"
+console.log(name)
+//=> "Raffy"
+//name = "Not Raffy" 
+//=> TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+console.log(drinkWater(12))
+function drinkWater(thirstLevel){
+// => "Man I sure am thirsty"
+console.log("Mann I sure am thirsty")
+thirstLevel -= 1
+// => "Ahh that hits the spot"
+console.log("Ahh that hits the spot")
+console.log(thirstLevel)
+// => 11
```

Comment:
Also, side note - remember to indent one tab in after you open up a block of code `{`